src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -34,3 +43,13 @@
     type: command
     short-summary: Show the details of Azure resource consumption, either as an invoice or within a billing period.
 """
+
+helps['consumption pricesheet'] = """
```

Comment:
Is "pricesheet" one word?  If not, strongly consider hyphenating "consumption price-sheet"

Reply:
Its not a single word, but according to the rest api published (https://docs.microsoft.com/en-us/rest/api/consumption/pricesheet/get ), its defined as a single word. So keeping with the pattern to avoid confusion.